packages/calypso-build/webpack.config.js

```diff
@@ -84,7 +84,6 @@ function getWebpackConfig(
 			rules: [
 				TranspileConfig.loader( {
 					workerCount,
-					configFile: path.join( __dirname, 'babel.config.js' ),
```

Comment:
Similar in spirit to #32045. This will make the build process pick up the `babel.config.js` from the CWD, rather than hard-wire to `@automattic/calypso-build`'s, allowing for customization (as we need for o2 blocks, and Jetpack).